Makefile

```diff
@@ -156,9 +156,12 @@ build-e2e-pex-files:
 build-engagement-view: ## Build website assets to include in grapl-web-ui
 	@echo "--- Building the engagement view"
 	$(ENGAGEMENT_VIEW_MAKE) build-code
+	TARGET_FRONTEND_DIR="${PWD}/src/rust/grapl-web-ui/frontend/"
```

Comment:
The `${PWD}` stuff isn't really needed; `src/rust/grapl-web-ui/frontend` is sufficient.

Also, the fact that there's a trailing `/` here means that the command on the next line is actually 
```shell
rm -rf ${PWD}/src/rust/grapl-web-ui/frontend//*
```
(for this reason, it's generally better to _not_ include trailing slashes in directory path variables)